### DISCUSSION TITLE
Dex Authenticating with Google: Not Found Invalid client_id ("").

### DISCUSSION THREAD

**Discussion by BenGee1996** [dexidp/dex#3453] [Q&A]
Hello,

I have got dex running on an AWS EKS Cluster and when navigating to the URL we have created, it displays a log in screen for our two providers okta and google.

We are deploying dex through the use of a helm chart.

When clicking google to login, I am greeted with the error 

**Not Found
Invalid client_id ("").**

This immediately makes me believe that the values.yaml file for the Dex helm chart is not recognising the client_id we have entered. 

However, our code matches that of the Dex documentation:

<img width="641" alt="Screenshot 2024-04-05 at 11 55 50" src="https://github.com/dexidp/dex/assets/107955272/b2276336-d406-4111-9f50-e33f08098e66">

Obviously I have removed the sensitive data, but could anyone assist with why the clientid is not being read from here? 

Just to be sure, I have amended other sections of the values.yaml file and confirmed it is taking effect, so the chart is definitely finding and using the values.yaml file.

**Comment by yashg-op** [comment]
Hi, were you able to solve this?

**Comment by yashg-op** [comment]
@BenGee1996

**Comment by franzudev** [comment]
It's happening also to me, anyone has the same problem?
